lab-geoff/greet.js

```diff
@@ -0,0 +1,3 @@
+exports.greet = function(name) {
```

Comment:
This file should be in a `lib/` directory, which is where your modules should all be placed.